### DISCUSSION TITLE
Difference between PF formulations

### DISCUSSION THREAD

**Discussion by AlexSacMorane** [idaholab/moose#18965] [Q&A Modules: Phase field] [answered]
Dear all,

I am investigating on  Phase Field formulations from [Moose documentation](https://mooseframework.inl.gov/modules/phase_field/Phase_Field_Equations.html) and I do not understand something.

I have tried the same set-up (2 grains defined by 2 differents conserved variables but with the same material properties). In the first case I use the following kernels (TimeDerivative, CahnHilliard, CHInterface) and in the second one I defined a chemical potential for each grain and I use (CoupleTimeDerivative, SplitCHParsed, SplitCHWRes). Results are not the same and I do not understand this because formulations should be equivalent. I have maybe missunderstood something or made a coding error.

You wil find in the attachment the 2 input files.

[PF_2grains_2Phases.txt](https://github.com/idaholab/moose/files/7253253/PF_2grains_2Phases.txt)
[PF_2grains_2Phases_Test.txt](https://github.com/idaholab/moose/files/7253255/PF_2grains_2Phases_Test.txt)

Thanks for all you can say.
I stay available. Regards.

Alexandre Sac--Morane

**Comment by AlexSacMorane** [comment]
Moreover, I do not understand why variables c_i skyrocket like this with time! I have consider the same set-up and material as the example _Conserved.i_ from the module _combined_. Is it because in my case the free energy depends on the 2 variables ?

**Reply by GiudGiud** [reply]
@laagesen @SudiptaBiswas @amjokisaari on phase field.

**Reply by AlexSacMorane** [reply]
Concerning this subject I found my error. The free energy was not well formulated.

**Comment by SudiptaBiswas** [comment, marked as answer]
Grains are usually represented by non-conserved order parameters. For simulating multiple grains, you should be looking at Allen-Cahn equation. Grain growth simulations would be a good place to start. If this doesn't help, please provide more information about the problem you are solving so you can point to right model to utilize.

**Reply by AlexSacMorane** [reply]
Ok I will change my simulation. I let you know. Thanks
